lib/loaders/loaders_with_authentication.js

```diff
@@ -0,0 +1,29 @@
+import { gravityLoaderWithAuthenticationFactory, impulseLoaderWithAuthenticationFactory } from "./api"
+const { IMPULSE_APPLICATION_ID } = process.env
+
+export default (accessToken, userID) => {
+  let impulseTokenLoader
+  const gravityLoader = gravityLoaderWithAuthenticationFactory(() => Promise.resolve(accessToken))
+  const impulseLoader = impulseLoaderWithAuthenticationFactory(() => impulseTokenLoader().then(data => data.token))
```

Comment:
It’s a bit of a chicken an egg problem, unless I define `impulseTokenLoader` in between `gravityLoader` and `impulseLoader`, because `impulseTokenLoader` depends on `gravityLoader`. I thought that it would improve readability if the various parts were defined in the same groupings, but maybe I was wrong?